Gh pages

SamPeck opened the pull request:
Updates to logos and images, plus include video in footer

simong committed:
Resize full logo on xs devices

simong committed:
Remove fronteer name

simong committed:
Remove small logo, vertically align main logo

simong committed:
Use new logo

Commit:
First version of Ally website

Commit:
Added links around vendor logos

Commit:
Accessibility improvements

Commit:
Reduced image sizes

Commit:
Increased size of footer text

Commit:
Improved placeholder text contrast

Commit:
Added webinar page

Commit:
Added email template for webinar

Commit:
Improved spacing in email template

Commit:
Add reminder for webinar 2

Commit:
Added Slide deck

Commit:
Added audio conversion

Commit:
Added Moodle documentation

Commit:
Added description

Commit:
Added new webinar

Commit:
Added email template for new webinar

Commit:
Update webinar invite

Commit:
Add webinar 4 page

Commit:
Added email for webinar 4

Commit:
Add correct link to web version

Commit:
Remove Sakai logo

Commit:
Updated webinar email

Added HTML images for webinar in Jan 2017

Commit:
Updated HTML email and deleted a file

Deleted registration page and updates to HTML Email

simong committed:
Merge pull request #2 from fronteerio/webinar

Updated webinar email

Commit:
Updated dates

Updated dates on the HTML email

simong committed:
Merge branch 'webinar' into gh-pages

* webinar:
  Updated dates

Commit:
Removed "Anne"

Removed placeholder name since Drip can replace that name

simong committed:
Merge branch 'webinar' into gh-pages

* webinar:
  Removed "Anne"

Commit:
Updates based on feedback

Language, alt tags and removal of “view in browser” link

simong committed:
Add location to the ical files

Commit:
Add slide deck to resources

Commit:
Add Ally Learn screens to resources

Commit:
Updated Drip tracking to page

Added Drip tracking URLs and JavaScript

Commit:
Update images and logos

Commit:
Updated webinar details

nicolaasmatthijs commented on index.html:
```diff
@@ -179,6 +179,9 @@ <h2 class="sr-only">Learning Management System integrations</h2>
     </div>
     <footer>
+        <div id="footer-video-container">
+                <iframe width="853" height="480" src="https://www.youtube.com/embed/PK-2DBIVDO8" frameborder="0" allowfullscreen></iframe>
```
@sam: Can you remove the title of the video? http://stackoverflow.com/questions/18342536/how-to-hide-the-bar-at-the-top-of-youtube-even-when-mouse-hovers-over-it explains how to do it

nicolaasmatthijs replied:
Also, the indentation is off here

nicolaasmatthijs commented on assets/img/logo-brand-mark.svg:
```diff
@@ -1,17 +1,16 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
The black of the hero image doesn't contrast very well with the black bar it's on.

nicolaasmatthijs replied:
Also, can you fix the padding/margin issue?

Commit:
Updates to website

Replaced hero image with laptop instead of monitor, fixed padding and
removed title from video.

nicolaasmatthijs commented:
@SamPeck : This is now asking for gh-pages to be merged into master. Going to close this.
